docs/audit_report/src/06_bibliography.rst

```diff
@@ -30,7 +30,9 @@
    Botan Test Report",
    Release TBD
 
-.. [BOTAN_GIT] https://github.com/randombit/botan/tree/3.1.1
+.. [BOTAN_GIT_300] https://github.com/randombit/botan/tree/3.0.0
+
+.. [BOTAN_GIT_310] https://github.com/randombit/botan/tree/3.1.0
```

Comment:
Nit: The reference should be to 3.1.1 (the production code base is the same, there was a race condition in the test framework).

```suggestion
.. [BOTAN_GIT_310] https://github.com/randombit/botan/tree/3.1.1
```